ndarbas08/duomenysZm..php

```diff
@@ -0,0 +1,8 @@
+<?php
+    $zmones = [
```

Comment:
Kur sis kintamasis naudojamas? Ar tai ne prie namu darbo?

Reply:
Vienoje užduotyje buvo riekalavimas masyvą turėti viename faile, funkciją - kitame. Funkcija turėjo paskaičiuoti amžių.

Reply:
Cia tos uzduoties sprendimo nematau.

Reply:
commit 'Uzduotys paskaitoje Nr04', failas - index.php

Reply:
Taip, o su situo pasalinta: https://github.com/RomanasCh/Romas/pull/8/commits/68cf72efb57ca241b9a0c6a398ad6a3e658d796f

Tai kam liko duomenys, jei pats kodas pasalintas?

Reply:
Tą patį index.php – failą naudoju visai sekai to pačio namų darbo užduotims paleisti. Uždaigęs užduotį commit‘inu visus užduoties failus, tame tarpe ir index.php. Su kiekvienu nauju commit‘u index.php failas perrašomas. Tam kad nagrinėti eilinę užduotį, reiktų pagal komentarą susirasti tarpinį commi‘tą. Paskutiniame commit‘e yra paskutinės užduoties failai ir atitinkamas index.php variantas.

Reply:
As suprantu kaip jus darote. As neziuriu commitu, as ziuriu jau rezultata, o rezultate tie duomenys jau niekur nenaudojami. Jei jie niekur nenaudojami tai ir turetu buti panaikinti, arba grazinkite uzduoties sprendima kuris dirbo su tais duomenimis.